src/renderer/components/ft-subscribe-button/ft-subscribe-button.vue

```diff
@@ -1,12 +1,83 @@
 <template>
-  <ft-button
-    :label="subscribedText"
-    class="subscribeButton"
-    background-color="var(--primary-color)"
-    text-color="var(--text-with-main-color)"
-    @click="handleSubscription"
-  />
+  <div
+    ref="subscribeButton"
+    class="ftSubscribeButton"
+    :class="{ dropdownOpened: isProfileDropdownOpen}"
+    @focusout="handleProfileDropdownFocusOut"
+  >
+    <div
+      class="buttonList"
+      :class="{ hasProfileDropdownToggle: isProfileDropdownEnabled}"
+    >
+      <ft-button
+        :label="subscribedText"
+        :no-border="true"
+        class="subscribeButton"
+        background-color="var(--primary-color)"
+        text-color="var(--text-with-main-color)"
+        @click="handleSubscription"
+      />
+      <ft-button
+        v-if="isProfileDropdownEnabled"
+        :no-border="true"
+        :title="isProfileDropdownOpen ? $t('Profile.Close Profile Dropdown') : $t('Profile.Open Profile Dropdown')"
+        class="profileDropdownToggle"
+        background-color="var(--primary-color)"
+        text-color="var(--text-with-main-color)"
+        :aria-expanded="isProfileDropdownOpen"
+        @click="toggleProfileDropdown"
+      >
+        <font-awesome-icon
+          :icon="isProfileDropdownOpen ? ['fas', 'angle-up'] : ['fas', 'angle-down']"
+        />
+      </ft-button>
+    </div>
+    <template v-if="isProfileDropdownOpen">
+      <div
+        tabindex="-1"
+        class="profileDropdown"
+      >
+        <ul
+          class="profileList"
+        >
+          <li
+            v-for="(profile, index) in profileDisplayList"
+            :id="'profile-' + index"
+            :key="'subscription-profile-' + index"
```

Comment:
Could you please change the id's and key to use the profile `_id` instead of the index, also they key can be simplified to just use the `_id`, as it is just used by Vue inside this component and isn't added to the DOM.

Reply:
Yes, will do; I admittedly was just copying the pattern used by the `ft-profile-selector`. I was unsure of whether this could cause conflict if we had other elements in the DOM with the same `key` values (e.g., if we changed the `ft-profile-selector` accordingly), but I'm pretty sure that is not a major issue after checking up on the documentation.

Reply:
For the `id` you should still use globally unique names, as those get placed into the DOM, but keys are only used for Vue processing in the scope of a component, so those don't have to have a unique prefix or anything like that.

Reply:
Hm... for some reason using the `_id` as key breaks `focusout`, even though it's populating properly... I am going to keep using the index + a specific prefix for the `id` and `key` (i.e., `subscription-profile-${index}` in light of this. I don't see any harm in it, and we do need unique prefixes as stated for the `id` if we're to have any other similar usage of `profile._id`s.